hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <iostream>
+#include "DomoticzHardware.h"
+
+class CThermostatHardware : public CDomoticzHardwareBase
+{
+private:
+
+	//thermostat available mode in string "OFF,ECO,COMFORT,AUTO," separated by comma, 
+	// and in the order from 0 to N and 0 = OFF 1 = ECO, 2 = CONFORT 3 = AUTO
```

Comment:
Use English instead of French
```suggestion
	// and in the order from 0 to N and 0 = OFF 1 = ECO, 2 = COMFORT 3 = AUTO
```